test/integration/host_js_test.rb

```diff
@@ -225,7 +225,7 @@ class HostJSTest < IntegrationTestWithJavascript
       page.find('#check_all').click
 
       # Ensure all hosts are checked
-      assert page.find('input.host_select_boxes').checked?
+      assert page.has_no_selector?('input.host_select_boxes:not(:checked)')
```

Comment:
has_no_unchecked_field seems to only work for a single field, as it calls the equivalent of `find_field` internally.  I'd need to specify an ID, name or label rather than the CSS class used now, but it'd mean losing the check for more than one host (if it happened for some reason).  I think it's probably better with the selector check.